### PR TITLE
Fixed #30507 -- Updated admin's jQuery to 3.4.1.

### PR DESCRIPTION
**Pull request by dulmandakh**

### CODE REVIEW

File: docs/ref/contrib/admin/index.txt [hunk 1]
```diff
@@ -2129,7 +2129,7 @@ jQuery
 Django admin JavaScript makes use of the `jQuery`_ library.
 
 To avoid conflicts with user-supplied scripts or libraries, Django's jQuery
-(version 3.3.1) is namespaced as ``django.jQuery``. If you want to use jQuery
+(version 3.4.1) is namespaced as ``django.jQuery``. If you want to use jQuery
```
Comment (felixxm) [inline docs/ref/contrib/admin/index.txt]: Please add also `versionchanged` note, e.g.
```
.. versionchanged:: 3.0

     jQuery was upgraded from 3.3.1 to 3.4.1. 
```

File: docs/releases/3.0.txt [hunk 2]
```diff
@@ -51,6 +51,8 @@ Minor features
   <django.contrib.admin.ModelAdmin.get_inlines>` method allows specifying the
   inlines based on the request or model instance.
 
+* jQuery is upgraded from version 3.3.1. to 3.4.1.
```
Comment (felixxm) [inline docs/releases/3.0.txt]: `3.3.1.` -> `3.3.1`

**Comment by dulmandakh** [comment]
@felixxm done, please review.

**Comment by felixxm** [comment]
buildbot, test on selenium.

**Comment by felixxm** [comment]
@dulmandakh Thanks! I rebased, squashed commits, and updated jQuery's license. I'm going to merged this in September.

**Comment by dulmandakh** [comment]
@felixxm thank you

**Comment by felixxm** [comment]
buildbot, test on selenium.

**Pull request merged**